kahuna/public/js/services/preset-label.js

```diff
@@ -0,0 +1,25 @@
+import angular from 'angular';
+
+var presetLabelService = angular.module('kahuna.services.presetLabel', []);
+
+presetLabelService.factory('presetLabelService',
+                            ['$window', function ($window) {
+
+    const presetLabelsKey = 'preset labels';
+
+    function getLabels() {
+        return JSON.parse($window.localStorage.getItem(presetLabelsKey));
+    }
+
+    function setLabels(presetLabelList) {
+        $window.localStorage.setItem(presetLabelsKey, JSON.stringify(presetLabelList));
+    }
+
+    return {
+        getLabels,
+        setLabels
+    };
+
+}]);
+
+export default presetLabelService;
```

Comment:
If we used [`Set`s](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set) here, adding and removing would be a little easier (we could do `addLabel`, `removeLabel` as opposed to doing the whole set).